main.py

```diff
@@ -50,15 +65,21 @@ def menu():
 def main():
     count_attemps = 1
     login_success = False
+
     while count_attemps <= 3:
         user_id = get_number_from_input("Enter your ID: ")
-        user_pass = get_number_from_input("Enter you password: ")
-        login_success = check_id_and_pin(user_id, user_pass)
-        if not login_success:
-            print(f"Something wrong with your ID or PASS. {count_attemps}/3. Try again")  # need not to TRY AGAIN at the last time
-            count_attemps += 1
+        user_pass = get_number_from_input("Enter your PASS: ")
+
+        if validation_credential(user_id, user_pass):
```

Comment:
maybe I need to use one more variable as we did with 'login_success'. But it won't be used more than 1 time